src/test/scala/org/broadinstitute/dsde/agora/server/dataaccess/acls/GcsAuthorizationSpec.scala

```diff
@@ -16,4 +16,13 @@ class GcsAuthorizationSpec extends FlatSpec {
     val entity = agoraBusiness.insert(testIntegrationEntity, "jcarey@broadinstitute.org")
     assert(agoraBusiness.findSingle(entity, Seq(entity.entityType.get), username) === entity)
   }
+
+  "Agora" should "check bucket write ACL when creating a new method in an existing workspace and fail if the user " +
+    "doesn't have access" in {
+    val thrown = intercept[AgoraAuthorizationException] {
+      val entity = agoraBusiness.insert(testIntegrationEntity, "ggrant@broadinstitute.org")
+      assert(agoraBusiness.findSingle(entity, Seq(entity.entityType.get), username) === entity)
+    }
+    assert(thrown != null)
+  }
 }
```

Comment:
This new test fails because of the order of execution - right?  Are these tests always guaranteed to executed in this order?